Source/WebCore/dom/TreeScope.h

```diff
@@ -135,7 +135,8 @@ class TreeScope {
     void addSVGResource(const AtomString& id, RenderSVGResourceContainer&);
     void addSVGResource(const AtomString& id, LegacyRenderSVGResourceContainer&);
     void removeSVGResource(const AtomString& id);
-    LegacyRenderSVGResourceContainer* svgResourceById(const AtomString& id) const;
+    RenderSVGResourceContainer* lookupSVGResourceById(const AtomString& id) const;
```

Comment:
This needs `ENABLE(LAYER_BASED_SVG_ENGINE)` guards -- not all ports compile it by default yet IIRC.

Reply:
Done.